shared/js/background/before-request.es6.js

```diff
@@ -214,39 +216,22 @@ function handleRequest (requestData) {
 function blockHandleResponse (thisTab, requestData) {
     const blockingEnabled = thisTab.site.isContentBlockingEnabled()
 
-    const tracker = trackers.getTrackerData(requestData.url, thisTab.site.url, requestData)
+    // Find the supported and enabled Click to Load rule actions for this tab.
+    const enabledRuleActions = new Set(
+        getDefaultEnabledClickToLoadRuleActionsForTab(thisTab).filter(ruleAction =>
+            !Object.prototype.hasOwnProperty.call(
+                thisTab.disabledClickToLoadRuleActions,
+                ruleAction
+            )
```

Comment:
Yea, if `thisTab.disabledClickToLoadRuleActions` started off as `Object.create(null)` instead of `{}` then this would be equivalent. I would have changed the Object to have no prototype, but I figured when it was read back from session storage that work would be undone.